mavlink/modules/flight_controller.py

```diff
@@ -101,3 +103,55 @@ def get_home_location(self,
             return False, None
 
         return True, location
+
+    def write_mission(self, commands: list[dronekit.Command]):
+        """
+        Writes a mission to the drone from a list of commands (will overwrite any previous missions)
+
+        Parameters
+        ----------
+        commands: list[dronekit.Command]
+        """
+
+        command_sequence = self.drone.commands
+        command_sequence.download()
+        command_sequence.wait_ready()
+        command_sequence.clear()
+        for command in commands:
+            command_sequence.add(command)
+
+        # Upload commands to drone
+        command_sequence.upload()
+
+
+    def create_land_command(self, latitude: float, longitude: float) -> "dronekit.Command":
```

Comment:
What this should do is actually overwrite the drone's current mission (and return whether it was successful or not).